Attendee.js

```diff
@@ -0,0 +1,39 @@
+// The Attendee class can be utilized to keep certain 
+// information of the attendee, a class for the user 
+// needed to be created in addition to the session class. 
+// The test for this JS class file utilizes Jest.IO
+class Attendee {
+    constructor(sessionId, screenName, timeLastPolled) {
+    /** @private {string} */
+        this.sessionId_ = sessionId;
```

Comment:
Weird spacing for the comments. The comment should start at the same time the actual variable starts. See https://google.github.io/styleguide/jsguide.html#jsdoc-property-comments